File: guilherme/Problema0438/Problema0438.py
def triangulo(a, b, c):
    x = 0
    if(a + b > c and a + c > b and b + c > a):
        if(a == b and b == c):
            x = 1
        elif(a == b and b != c or a == c and a != b or b == c and c != a):
            x = 2
        elif(a != b and a != c and b != c):
            x = 3
    return x

print("Digite o primeiro lado: ") 
a = int(input()) 
print("Digite o segundo lado: ") 
b = int(input()) 
print("Digite o terceiro lado: ") 
c = int(input()) 
if(triangulo(a, b, c) == 0):
    print("Não é um triângulo!") 
elif(triangulo(a, b, c) == 1):
    print("Triângulo Equilátero!") 
elif(triangulo(a, b, c) == 2):
    print("Triângulo Isóceles!") 
elif(triangulo(a, b, c) == 3):
    print("Triângulo Escaleno!") 

File: guilherme/Problema0438/Problema0438.java
import java.util.Scanner;

public class Problema0438{
    static Scanner input = new Scanner(System.in);

    public static int triangulo(int a, int b, int c){
        int x = 0;
        if(a + b > c && a + c > b && b + c > a){
            if(a == b && b == c){
                x = 1;
            }else if(a == b && b != c || a == c && a != b || b == c && c != a){
                x = 2;
            }else if(a != b && a != c && b != c){
                x = 3;
            }
        }
        return x;
    }

    public static void main(String[] args) {
        int a, b, c;
        System.out.println("Digite o primeiro lado: ");
        a = Integer.valueOf(input.nextLine());
        System.out.println("Digite o segundo lado: ");
        b = Integer.valueOf(input.nextLine());
        System.out.println("Digite o terceiro lado: ");
        c = Integer.valueOf(input.nextLine());
        if(triangulo(a, b, c) == 0)
            System.out.println("Não é um triângulo!");
        else if(triangulo(a, b, c) == 1)
            System.out.println("Triângulo Equilátero!");
        else if(triangulo(a, b, c) == 2)
            System.out.println("Triângulo Isóceles!");
        else if(triangulo(a, b, c) == 3)
            System.out.println("Triângulo Escaleno!");
    }
}